testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/util/DeltaLakeTestUtils.java

```diff
@@ -29,14 +29,12 @@ private DeltaLakeTestUtils() {}
 
     public static String getDatabricksRuntimeVersion()
     {
-        QueryResult result = onDelta().executeQuery("SELECT java_method('java.lang.System', 'getenv', 'DATABRICKS_RUNTIME_VERSION')");
-        return firstNonNull((String) result.row(0).get(0), "unknown");
+        return firstNonNull((String) onDelta().executeQuery("SELECT java_method('java.lang.System', 'getenv', 'DATABRICKS_RUNTIME_VERSION')").getOnlyValue(), "unknown");
```

Comment:
it's not necessarily easier to read this form.

Reply:
broke the statement in two lines